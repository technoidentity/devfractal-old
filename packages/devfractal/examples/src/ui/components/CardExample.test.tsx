import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { CardExample } from './CardExample'

test('Card Example', () => {
  const component: ReactTestRenderer = renderer.create(<CardExample />)
  expect(component.toJSON()).toMatchSnapshot()
})
