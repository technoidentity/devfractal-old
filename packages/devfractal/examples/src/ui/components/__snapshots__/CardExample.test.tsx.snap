// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Example 1`] = `
Array [
  <section
    className="section"
  >
    <div
      className="card"
    >
      <header
        className="card-header"
      >
        <p
          className="card-header-title is-centered"
        >
          Card
        </p>
      </header>
      <div
        className="card-image"
      >
        <figure
          className="image is-4by3"
        >
          <img
            className=""
            src="https://bulma.io/images/placeholders/1280x960.png"
          />
        </figure>
      </div>
      <div
        className="card-content"
      >
        <article
          className="media"
        >
          <figure
            className="media-left"
          >
            <figure
              className="image is-48x48"
            >
              <img
                className=""
                src="https://bulma.io/images/placeholders/96x96.png"
              />
            </figure>
          </figure>
          <div
            className="media-content"
          >
            <h1
              className="title is-4"
            >
              John Smith
            </h1>
            <h1
              className="subtitle is-6"
            >
              @johnsmith
            </h1>
          </div>
        </article>
        <div
          className="content"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus nec iaculis mauris.
          <a
            href="#!"
          >
            @bulmaio
          </a>
          . 
          <a
            href="#!"
          >
            #css
          </a>
          <a
            href="#!"
          >
            #responsive
          </a>
          <br />
          <time
            dateTime="2016-1-1"
          >
            11:09 PM - 1 Jan
          </time>
        </div>
      </div>
      <footer
        className="card-footer"
      >
        <div
          className="card-footer-item"
        >
          <a
            href="#!"
          >
            save
          </a>
        </div>
        <div
          className="card-footer-item"
        >
          <a
            href="#!"
          >
            edit
          </a>
        </div>
        <div
          className="card-footer-item"
        >
          <a
            href="#!"
          >
            delete
          </a>
        </div>
      </footer>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="card"
    >
      <header
        className="card-header"
      >
        <p
          className="card-header-title"
        >
          Component
        </p>
        <a
          className="card-header-icon"
        >
          <span
            className="icon"
          >
            <svg
              aria-hidden="true"
              className="svg-inline--fa fa-angle-down fa-w-10 "
              data-icon="angle-down"
              data-prefix="fas"
              focusable="false"
              role="img"
              style={Object {}}
              viewBox="0 0 320 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M143 352.3L7 216.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.2 9.4-24.4 9.4-33.8 0z"
                fill="currentColor"
                style={Object {}}
              />
            </svg>
          </span>
        </a>
      </header>
      <div
        className="card-content"
      >
        <div
          className="content"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus nec iaculis mauris.
          <a
            href="#!"
          >
            @bulmaio
          </a>
          . 
          <a
            href="#!"
          >
            #css
          </a>
          <a
            href="#!"
          >
            #responsive
          </a>
          <br />
          <time
            dateTime="2016-1-1"
          >
             11:09 PM - 1 Jan 2016
          </time>
        </div>
      </div>
      <footer
        className="card-footer"
      >
        <div
          className="card-footer-item"
        >
          <a
            href="#!"
          >
            Save
          </a>
        </div>
        <div
          className="card-footer-item"
        >
          <a
            href="#!"
          >
            Edit
          </a>
        </div>
        <div
          className="card-footer-item"
        >
          <a
            href="#!"
          >
            Delete
          </a>
        </div>
      </footer>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="card"
    >
      <div
        className="card-content"
      >
        <h1
          className="title"
        >
          “There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.”
        </h1>
        <h1
          className="subtitle"
        >
          Jeff Atwood
        </h1>
      </div>
      <footer
        className="card-footer"
      >
        <div
          className="card-footer-item"
        >
          <span>
            View on 
            <a
              href="#!"
            >
               Twitter
            </a>
          </span>
        </div>
        <div
          className="card-footer-item"
        >
          <span>
            Share on 
            <a
              href="#!"
            >
              Facebook
            </a>
          </span>
        </div>
      </footer>
    </div>
  </section>,
]
`;
